tests/tests.go

```diff
@@ -828,14 +827,57 @@ func getAggregatedTaskStatus(issue *api.Issue) (api.TaskStatus, error) {
 			case api.TaskCanceled:
 				return api.TaskCanceled, nil
 			case api.TaskRunning:
-				running = true
```

Comment:
We can refactor this big switch case to only handle `if task.Status == api.TaskFailed { ... } else { return task.Status, nil }

Reply:
Done